prelude/runtime/free.c

```diff
@@ -39,7 +39,7 @@ void sk_free_obj(sk_stack_t* st, char* obj) {
 
   // Check if we are dealing with a string
   if (SKIP_is_string(obj)) {
-    size_t memsize = get_sk_string(obj)->size;
+    size_t memsize = get_sk_string(obj)->size + 1;
```

Comment:
This isn't always true, is it?

Reply:
Did you have something specific in mind? My thinking was that for static strings their size is rounded up to a multiple of 8, but those shouldn't be given to free anyhow, and all the other strings are allocated by `sk_string_alloc` in `string.c` which does allocate this number of bytes.

Reply:
Now that you unconditionally add the extra byte, that's good